other_types_of_ownership/multi_sig.md

```diff
@@ -80,9 +80,9 @@ Transaction bobSigned =
         .SignTransaction(aliceSigned);
 ```  
 
-![](https://i.imgur.com/xaxR9MV.png)  
+![](../assets/bobSigned.png)  
 
-Now, Bob and Alice can combine their signature into one transaction. This transaction will then be valid in terms of it's signature as Bob and Alice have provided two of the signatures from the three owner signatures that were initially provided. The requirements of the 'two-of-three' multi sig have therefore been met.
+Now, Bob and Alice can combine their signature into one transaction. This transaction will then be valid, because two (Bob and Alice) signatures were used from the three (Alice, Bob and Satoshi) signatures that were initially provided. The requirements of the 'two-of-three' multisig have therefore been met. If it wasn't be the case the network would not accept this transaction, because it rejects all unsigned or partially signed transactions by default.
```

Comment:
```
because it rejects all unsigned or partially signed transactions by default
```

The correct way would be `(the network) rejects all invalid transactions.` This is not "by default". All nodes will reject an invalid transactions.